src/plugins/content_management/server/content_client/content_client.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import type { StorageContext } from '../core';
+import { ContentCrud } from '../core/crud';
+import type { IContentClient } from './types';
+
+interface Context<T = unknown> {
+  crudInstance: ContentCrud<T>;
+  storageContext: StorageContext;
+}
+
+const secretToken = Symbol('secretToken');
```

Comment:
Note to self: this never gets stored. I had to check because that could pose an issue if we ever handle a token that was created in a separate Kibana instance